templates/scripts/create_rpc_node_compose_file.sh

```diff
@@ -106,7 +106,6 @@ services:
       "--base-path", "/var/subspace",
       "--state-pruning", "archive",
       "--blocks-pruning", "archive",
-      "--sync", "full",
```

Comment:
This is not correct for RPC node.

RPC node needs to sync from genesis to have all the contents in it. Syncing with Snap sync will not review any blocks, but it will not have older blocks before Snap sync either.

Reply:
Ok, i've enabled full sync on the RPC nodes only